components/event_dispatcher.rst

```diff
@@ -303,7 +303,7 @@ each listener of that event::
 
     // creates the OrderPlacedEvent and dispatches it
     $event = new OrderPlacedEvent($order);
-    $dispatcher->dispatch(OrderPlacedEvent::NAME, $event);
+    $dispatcher->dispatch($event, OrderPlacedEvent::NAME);
```

Comment:
Maybe we should also drop the name too as it is not needed anymore.

Reply:
This is a hot fix.  If we drop the name, we go out from page context

Reply:
I agree with @shude. Also, I'm going to submit soon a total revamp of the events docs ... so this will be included in that revamp. Thanks.